inc/saiswitch.h

```diff
@@ -349,7 +349,14 @@ typedef enum _sai_switch_attr_t
     /** Nonvolatile storage required by both SAI and NPU in KB [sai_uint64_t]
      * Will be 0 for SAI_RESTART_TYPE_NONE */
     SAI_SWITCH_ATTR_NV_STORAGE_SIZE,
-
+    
+    /** Whether action list is mandatory for table creation [bool] 
+     * Default [false]*/
+    SAI_SWITCH_ATTR_ACL_TABLE_ACTION_LIST_MANDATORY,
+
+    /** List of actions supported [sai_u32_list_t] 
+     * List of all supported actions will be returned */
+    SAI_SWITCH_ATTR_ACL_TABLE_ACTION_LIST,
```

Comment:
@atitjain @AnbalaganNatchimuthu "Do we assume mandatory to be same for all stages etc", looks like it does. However, this may not be a valid assumption here. In this case, for each stage the SAI supports, we need to know whether the action list is mandatory or not.